.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+## Describe the problem
+
+## Describe the solution
+
+## Risk & Testing considerations
+What is the risk level of this change (Low / Medium / High)? For Medium / High please elaborate.
+
+Which modules are impacted by this change?
+
+Which integration tests and or unit tests are applicable for it? This could be a whole suite of Python tests or specific tests.
+
+Has new tests been written which exercises the relevant code paths? If yes, please identify this via the code coverage report. If not, explain why.
+
+What other testing is relevant for this change? This could include: stress testing, MongoDB patch builds, etc.
+
+## Checklist before requesting a review
+
+- [ ] I have performed a self-review of my code.
```

Comment:
This is looking very good, is the plan that this will be automatically added to the PR description?

Reply:
Yep. I expect once this PR gets merged, the PR template will automatically pop up when new PRs are raised.

Reply:
According to [Github documentation](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates#pull-request-templates)
> You must create templates on the repository's default branch. Templates created in other branches are not available for collaborators to use.

which makes it hard to verify the effect without merging this PR. 

Instead, I used a test repo (wiredtiger-mirror) and merged the current template change into the develop branch, and created a test PR to show the effect. https://github.com/wiredtiger/wiredtiger-mirror/compare/wt-13422-test-pr-template?expand=1